mico-core/src/main/java/io/github/ust/mico/core/MicoCoreApplication.java

```diff
@@ -55,4 +78,30 @@ public static void main(String[] args) {
     public RestTemplate restTemplate(RestTemplateBuilder builder) {
         return builder.build();
     }
+
+    public void onApplicationEvent(final ApplicationReadyEvent event) {
+        try {
+            MicoService kafkaFaasConnector = gitHubCrawler.crawlGitHubRepoLatestRelease(kafkaConfig.getKafkaFaasConnectorUrl());
```

Comment:
We need to skip this part when running unit tests. Travis cannot test anything that uses our GitHub crawler and when unit testing this code will probably be executed many many times!
To properly test the code refactor it out in a testable method and test this method with a unit test skipped by travis.